### PR TITLE
New master

### PR DESCRIPTION
**Pull request by Ayopal**
ES6 for of method was added

### CODE REVIEW

File: arrowFunctions.js [hunk 1]
```diff
@@ -4,6 +4,8 @@
  * Parameter(s):
  * nums: An array of numbers.
  */
+
+ var nums = [1, 2, 3, 4, 5];
```
Comment (fobabs) [inline arrowFunctions.js]: @Ayopal This part isn't necessary for the solution. It is for your personal use.

File: arrowFunctions.js [hunk 2]
```diff
@@ -13,9 +15,32 @@ function modifyArray(nums) {
...
-      nums[i] * 2;
+     nums[i] *= 2; 
     else
-      nums[i] * 3;
+     nums[i] *= 3;
   }
-  return nums;
-}
\ No newline at end of file
+   return nums;
```
Comment (fobabs) [inline arrowFunctions.js]: @Ayopal Use proper indentation here.

File: arrowFunctions.js [hunk 3]
```diff
@@ -13,9 +15,32 @@ function modifyArray(nums) {
...
+} 
+
+}
+
+
+var p = modifyArray6(nums);
+console.log(p);
+
+
+
```
Comment (fobabs) [inline arrowFunctions.js]: @Ayopal This code snippet doesn't return an array and again does not use arrow functions.

File: arrowFunctions.js (hunk 1 above)
Comment (Ayopal) [reply arrowFunctions.js]: line removed

File: arrowFunctions.js (hunk 3 above)
Comment (Ayopal) [reply arrowFunctions.js]: code removed and indentation adjusted;

Please kindly advice on a good snippet/plug-in to use for auto-indent

**Review by Ayopal — COMMENTED** [review]
Please peruse all the codes again as I await your merging soon

File: arrowFunctions.js (hunk 2 above)
Comment (Ayopal) [reply arrowFunctions.js]: corrected.  Kindly advice on a snippet to use for auto indent

**Pull request closed without merging**